gogio/androidbuild.go

```diff
@@ -461,11 +463,22 @@ func exeAndroid(tmpDir string, tools *androidTools, bi *buildInfo, extraJars, pe
 			android:theme="@style/Theme.GioApp"
 			android:configChanges="screenSize|screenLayout|smallestScreenSize|orientation|keyboardHidden"
 			android:windowSoftInputMode="adjustResize"
+			{{- if .Schemes}}
+			android:launchMode= "singleInstance" 
+			{{- end}}
```

Comment:
Why not unconditionally set the launch mode to "singleInstance"? I don't think Gio currently supports multiple instances of `GioActivity`/`GioView` in the same process, right?

Reply:
I don't know, I just want to be more _conservative_ and avoid changes for non-users of `scheme`. The `singleInstance` is required for `scheme`, otherwise it renders a blank screen. 

I can change to use `singleInstance` in all cases.